portfolio/src/main/webapp/script.js

```diff
@@ -47,16 +47,16 @@ function loadComments() {
   fetch('/data').then(response => response.json()).then(allComments => {
     const commentContainer = document.getElementById('comment-container');
     allComments.forEach(commentText => {
-      const commentItem = createCommentItem(commentText.text);
+      const commentItem = createCommentItem(commentText.text, commentText.time);
       commentContainer.appendChild(commentItem);
     });
   });
 }
 
 // Helper function to instantiate each comment.
-function createCommentItem(text) {
+function createCommentItem(text, time) {
   const commentItem = document.createElement('div');
-  commentItem.innerText = text;
+  commentItem.innerText = "\"" + text + "\" sent at " + time;
```

Comment:
It might be nice to format the time in milliseconds into something more easily human readable.

Perhaps `new Date(time_millis).toString()` ? Although in this context the `toString()` part may not be required.

Reply:
Good idea, I changed this to use something called `SimpleDateFormat` which looks a lot nicer.